arangod/Pregel/Status/ConductorStatus.h

```diff
@@ -33,34 +34,45 @@
 #include <Cluster/ClusterTypes.h>
 
 #include <Pregel/Common.h>
-#include <Pregel/Status/WorkerStatus.h>
+#include <Pregel/Status/Status.h>
 
 namespace arangodb::pregel {
 
-struct ConductorStatus {
-  TimeStamp timeStamp = std::chrono::system_clock::now();
-  std::unordered_map<ServerID, WorkerStatus> workers;
+struct AccumulatedConductorStatus {
+  Status status;
+  std::unordered_map<ServerID, Status> workers;
+  bool operator==(AccumulatedConductorStatus const&) const = default;
+};
+
+template<typename Inspector>
+auto inspect(Inspector& f, AccumulatedConductorStatus& x) {
```

Comment:
Maybe, better names for `f` and `x`.

Reply:
These functions are usually very short, and it is clear from the type of the parameter what it is. 

Making the names longer might make the inspector functions less readable, because the inspector is used all over the place, as is the struct.

Also, I don't think deviating from the current style across the ArangoDB codebase is a good idea. 

Don't call a variable `struct` though ;)